sdcm/db_stats.py

```diff
@@ -371,6 +373,9 @@ def get_scylla_versions(self):
                         versions[package.replace('-enterprise', '')] = {'version': match.group(2),
                                                                         'date': match.group(4),
                                                                         'commit_id': match.group(5)}
+                    LOGGER.info("versions[package.replace('-enterprise', '')]=%s", versions[package.replace('-enterprise', '')])
+                    if build_id_output:
+                        versions[package.replace('-enterprise', '')]['build_id'] = build_id_output
```

Comment:
why do you need to add build -id for each package? may be it is need only for scylla package?

Reply:
Also you try to add  build-id  for key which is not yet created. 
in versions_output, first line is scylla
while in loop, first is scylla-jmx and match is not happened, and therefore the versions[scylla-jmx] is not created, and you try to add versions[scylla-jxm][build-id], these cause an error

Reply:
ah, if it eventually wasn't yet created...
so, in the latest version i'm adding it only to `scylla-server`, but also i might need to "protect" it against it eventually not being created...